CONTRIBUTING.md

```diff
@@ -11,6 +11,27 @@ Because this package uses `internal` packages, all fork development has to happe
 7. push to the branch: `git push fork my-new-feature`
 8. create a new Pull Request.
 
+### Local web-based godoc
```

Comment:
I feel that "web-based" is a bit misleading here. Also, this title doesn't quite align with the next heading "Running Tests". So, perhaps this might be better:

```suggestion
### Previewing Godoc Locally
```

Reply:
done